cpu/cc26xx-cc13xx/rf-core/ieee-mode.c

```diff
@@ -908,16 +908,8 @@ transmit(unsigned short transmit_len)
   cmd.startTime = 0;
   cmd.startTrigger.triggerType = TRIG_NOW;
 
-  /* XXX: there seems to be no function that gets interrupt state w/o changing it */
-  interrupts_enabled = ti_lib_int_master_disable();
```

Comment:
`interrupts_enabled` is now an unused variable and generates a warning. Travis will fail. The definition needs removed.